From c138400edd7957e10ff9ed11b6f6887586b25e3b Mon Sep 17 00:00:00 2001
From: Neil Fuller <nfuller@google.com>
Date: Thu, 4 Oct 2018 22:37:18 +0100
Subject: [PATCH 02/25] Swap libcore annotations for framework versions

Swap libcore nullability annotations for android.annotation
equivalents.

Bug: 113148576
Test: build
Change-Id: Ie571062779e9bdf3890f6fcbd99be905aaf1adf9
---
 .../java/com/android/server/pm/dex/ArtManagerService.java     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/core/java/com/android/server/pm/dex/ArtManagerService.java b/services/core/java/com/android/server/pm/dex/ArtManagerService.java
index 21daa39e05e0..4b471b2b65e5 100644
--- a/services/core/java/com/android/server/pm/dex/ArtManagerService.java
+++ b/services/core/java/com/android/server/pm/dex/ArtManagerService.java
@@ -16,6 +16,8 @@
 
 package com.android.server.pm.dex;
 
+import android.annotation.NonNull;
+import android.annotation.Nullable;
 import android.annotation.UserIdInt;
 import android.app.AppOpsManager;
 import android.content.Context;
@@ -57,8 +59,6 @@ import dalvik.system.DexFile;
 import dalvik.system.VMRuntime;
 
 import libcore.io.IoUtils;
-import libcore.util.NonNull;
-import libcore.util.Nullable;
 
 import java.io.File;
 import java.io.FileNotFoundException;
-- 
2.17.1

